package.json

```diff
@@ -19,5 +19,8 @@
     "bugs": {
         "url": "https://github.com/migalabs/eth-metadashboard/issues"
     },
-    "homepage": "https://github.com/migalabs/eth-metadashboard#readme"
-}
\ No newline at end of file
+    "homepage": "https://github.com/migalabs/eth-metadashboard#readme",
+    "dependencies": {
+        "framer-motion": "^10.16.10"
```

Comment:
@Artentii never add dependencies in the root directory, you should add it in the client driectory

Reply:
Okey, I'll bare that in mind